### DISCUSSION TITLE
Reihenhäuser

### DISCUSSION THREAD

**Discussion by biketeur** [streetcomplete/StreetComplete#5958] [Ideas]
Seit einiger Zeit werden einzelne Reihenhäuser nicht nur mit 'building=house' getaggt, sondern zusätzlich mit 'house=terraced'.
Sind beide Werte bei einem Gebäude gesetzt, erkennt dies SC und gibt folgerichtig Reihenhaus als Gebäudetyp aus.
Frage: wäre es nicht sinnvoll, wenn SC diese neue Konvention auch bei der Erfassung/Eingabe unterstützten würde?

**Comment by westnordost** [comment]
Ungern, weil die Liste der auswählbaren Gebäudetypen schon soooo lang und überfrachtet ist. Ich finde gut dass sich das `house=*` etabliert, weil das eben bedeutet, dass es **nicht** notwendig ist, in dem Formular alle möglichen Sub-Typen eines Einfamilienhauses auch noch als Selektionsmöglichkeiten anzubieten.
